src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -282,4 +282,19 @@ In order to debug this project, add an executable project to this solution which
   <data name="OutputWindowPaneTitle" xml:space="preserve">
     <value>Project</value>
   </data>
+  <data name="PartialSupportedDotNetCoreProject" xml:space="preserve">
+    <value>Visual Studio 2017 15.7 or newer is recommended for .NET Core 2.1 projects.</value>
```

Comment:
I thinking branding recommendations are to put "version" between 2017 and 15.7.

Reply:
Good point